src/main/resources/static/eiffel-rabbit-mq-post-receive-hook.soy

```diff
@@ -19,12 +19,21 @@
   {/call}
 
   {call aui.form.textareaField}
-    {param id: 'rabbit.exchange' /}
```

Comment:
We usually have exchange name before its type. Nothing is wrong with placing type before the name, but I would keep the name first for the sake of consistency